mkdocs.yml

```diff
@@ -269,15 +244,15 @@ markdown_extensions:
 
 
 ### Customization
-extra:
+# extra:
   # version:
   #   default: latest
   #   provider: mike
   # disqus: !ENV REPO_OWNER
-  alternate:
-  - name: en - English
-    link: /
-    lang: en
+  #alternate:
+  #- name: en - English
+  #  link: /
+  #  lang: en
```

Comment:
:+1: Since, the documentation is available only in English, this is not necessary for now. But if you want to make documentation available in other languages, then this setting could be helpful.

Reply:
If that happens (unlikely) we can add it.